tracer/src/Datadog.Tracer.Native/cor_profiler.cpp

```diff
@@ -2643,6 +2681,68 @@ HRESULT CorProfiler::RewriteForTelemetry(const ModuleMetadata& module_metadata,
     return hr;
 }
 
+HRESULT CorProfiler::RewriteIsManualInstrumentationOnly(const ModuleMetadata& module_metadata, ModuleID module_id)
+{
+    //
+    // *** Get Instrumentation TypeDef
+    //
+    mdTypeDef instrumentationTypeDef;
+    HRESULT hr = module_metadata.metadata_import->FindTypeDefByName(instrumentation_type_name.c_str(),
+                                                                    mdTokenNil, &instrumentationTypeDef);
+    if (FAILED(hr))
+    {
+        Logger::Warn("Error rewriting IsManualInstrumentationOnly on getting Instrumentation TypeDef");
```

Comment:
Shouldn't this be an error? If this fails (type / method not found) something nasty must have happened...

Reply:
Probably 🤷‍♂️ But 
- It's going to be logged in both cases anyway
- Neither one ends up in telemetry logs
- It doesn't break the overall tracer, just the manual instrumentation
- It's what we use for other rewriting currently

So, meh? 😃